packages/xo-server-test/connect.mjs

```diff
@@ -1,7 +1,16 @@
+import appConf from 'app-conf'
 import Xo from 'xo-lib'
 const XoConnection = Xo.default
 
-const SERVER_URL = process.env.SERVER_URL || 'http://127.0.0.1:80'
+const xoServerHttpConfig = (
+  await appConf.load('xo-server', {
+    appDir: new URL('..', import.meta.url).pathname,
+    ignoreUnknownFormats: true,
+  })
+).http.listen[0]
+const port = xoServerHttpConfig.port || 80
```

Comment:
I think the default value could be pulled from the environment, to ease the test automation later 
```js
const port = xoServerHttpConfig.port || process.env.SERVER_URL || 80
```
And something similar for the url

Reply:
This is what was done in xo-server-test. 
@julien-f suggested me to use app-conf instead.

In the case of test automation we would also receive these parameters from the configuration, right?

Reply:
we'll add the option later if needed